packages/react-server/src/ReactFlightServer.js

```diff
@@ -4077,7 +4091,6 @@ export function halt(request: Request, reason: mixed): void {
     // We have tasks to abort. We'll emit one error row and then emit a reference
     // to that row from every row that's still remaining.
     if (abortableTasks.size > 0) {
-      request.pendingChunks++;
```

Comment:
this was a bug I landed earlier. it prevents the flush from ever completing because we don't actually have a new pending chunk here